jenkins/node-dockerized.sh

```diff
@@ -37,18 +37,19 @@ mkdir -p "${HOST_ARTIFACTS_DIR}"
 
 # default to go version 1.6 image tag
 # TODO (krzyzacy) Make one image per kubekins-image version.
-NODEIMAGE_TAG="v20161219-40a194e"
+NODEIMAGE_TAG="v20161219-b93fda0"
 
 docker run --rm=true \
   --privileged=true \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v "${REPO_DIR}":/go/src/k8s.io/kubernetes \
   -v "${WORKSPACE}/_artifacts":/workspace/artifacts \
   -v /etc/localtime:/etc/localtime:ro \
+  ${GOOGLE_APPLICATION_CREDENTIALS:+-v "${GOOGLE_APPLICATION_CREDENTIALS}:/service-account.json:ro"} \
```

Comment:
I think you also need to set the `GOOGLE_APPLICATION_CREDENTIALS` env var for the line in `node-runner.sh` to use this.